Update console intro instructions (#25748)
mdn/content: files/en-us/web/javascript/guide/introduction/index.md

## Getting started with JavaScript

To get started with JavaScript, all you need is a modern web browser. Recent versions of [Firefox](https://www.mozilla.org/en-CA/firefox/new/), [Chrome](https://www.google.com/chrome/index.html), [Microsoft Edge](https://www.microsoft.com/en-us/edge), and [Safari](https://www.apple.com/safari/) all support the features discussed in this guide.

> ### The JavaScript Console

Reviewer: I suggest removing this heading, because the content until the end of page is all about "the JavaScript console".

A very useful tool for exploring JavaScript is the JavaScript Console (sometimes called the Web Console, or just the console): this is a tool which enables you to enter JavaScript and run it in the current page.

The screenshots here show the [Firefox Web Console](https://firefox-source-docs.mozilla.org/devtools-user/web_console/), but all modern browsers ship with a console that works in a similar way.

### Opening the console

The exact instructions for opening the console vary from one browser to another:

- [Opening the console in Firefox](https://firefox-source-docs.mozilla.org/devtools-user/web_console/#opening-the-web-console)
- [Opening the console in Chrome](https://developer.chrome.com/docs/devtools/open)

> - [Opening the console in Microsoft Edge](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/)

Reviewer: Is there a Safari link? (Maybe not, just confirming)

Author: No, as said in the description, I couldn't find useful devtools docs for Safari 🤷

Reviewer: Sorry for missing that sentence! I didn't find any official documentation either, apart from https://support.apple.com/guide/safari/use-the-developer-tools-in-the-develop-menu-sfri20948/mac. I'm okay to omit it because you probably found this link as well.

console.log(eval("3 + 5"));
```

### Multi-line input in the Console

By default, if you press <kbd>Return</kbd> after entering a line of code, then the string you typed is executed. To enter multi-line input:

> - If the string you typed was incomplete (for example, you typed `function foo() {`) then the console will treat <kbd>Return</kbd> as a return, and let you type another line.
> - If you hold down <kbd>Shift</kbd> while pressing <kbd>Return</kbd>, then the console will treat this as a return, and let you type another line.

Reviewer: ```suggestion
- If the string you typed was incomplete (for example, you typed `function foo() {`) then the console will treat <kbd>Return</kbd> as a line break, and let you type another line.
- If you hold down <kbd>Shift</kbd> while pressing <kbd>Return</kbd>, then the console will treat this as a line break, and let you type another line.
```

- In Firefox only, you can activate [multi-line input mode](https://firefox-source-docs.mozilla.org/devtools-user/web_console/the_command_line_interpreter/index.html#multi-line-mode), in which you can enter multiple lines in a mini-editor, then run the whole thing when you are ready.

> To get started with writing JavaScript, open the Firefox Web Console in multi-line mode, and write your first "Hello world" JavaScript code:

Reviewer: ```suggestion
To get started with writing JavaScript, try entering your first "Hello world" JavaScript code:
```

Author: If you want this change we have to make more changes, because the line just below says:

> Press <kbd>Cmd</kbd>+<kbd>Enter</kbd> or <kbd>Ctrl</kbd>+<kbd>Enter</kbd> (or click the **Run** button) to watch it unfold in your browser!

Also, "entering" here has to have a specific meaning, either "copy/pasting" or "typing in and shift-returning". If someone reads "entering" and "typing a line at a time and pressing return", it isn't going to work.

I can change it to "copy/paste" which probably makes more sense than asking someone to type it all in.

Reviewer: My main reason for edit was to get rid of "Firefox Web Console in multi-line mode" because you can enter this code in other consoles as well, and people may not be necessarily using Firefox. Now I see how "enter" may be ambiguous. "paste the "Hello world" JavaScript code" does sound better 👍
